contributed/plots/README copy.md

```diff
@@ -0,0 +1,5 @@
+# Plot Descriptions
+
+## Correlation Functions from TXPipe with CosmoDC2_small
+
+These plots were made by running the cosmoDC2_v1.0_small catalog through TXPipe.  TXPipe is a pipeline to perform 3x2 pt. analysis or the joint analysis of galaxy clustering, cosmic shear and galaxy-galaxy lensing on DC2 data.  The plots included show output of the pipleline of the three correlation functions for the correlation of galaxy positions, galaxy shears and the cross correlation of galaxy position and shears.  A mock metacal catalog was generated from the input catalog using the stage input_cats.py in the pipeline that simulates noise and shear response. 
```

Comment:
One thing missing in this blurb about the plots is attribution - @ellongley, you should take credit for your work by saying in the README that you who did the work to make these plots (and perhaps put your GitHub username so people with questions can easily ping you).  In fact it might also be nice to put a link to this PR thread for people who want more info.

Reply:
Oh also, assuming others' plots are also going to go in the directory, your blurb should specifically list the filenames of the associated plots.

Reply:
Thank you so much for the feedback! There are three new commits addressing these suggestions.